axelor-project/src/main/resources/views/TeamTask.xml

```diff
@@ -112,11 +112,6 @@
 				<field name="totalPlannedHrs" readonly="true" colSpan="4">
 				<viewer><![CDATA[
                    	<span>{{record.totalPlannedHrs}} </span><span x-translate>hours</span>
-       			]]></viewer>
-				</field>
-				<field name="totalRealHrs" readonly="true" colSpan="4">
-				<viewer><![CDATA[
-                   	<span>{{record.totalRealHrs}} </span><span x-translate>hours</span>
        			]]></viewer>
 				</field>
 				<field name="budgetedTime" colSpan="4">
```

Comment:
Add spacer with colSpan="4" after budgetedTime
as currently, the view is getting improper